upstairs/src/lib.rs

```diff
@@ -9029,6 +9029,13 @@ pub struct Guest {
      */
     bw_tokens: Mutex<usize>, // bytes
     bw_limit: Option<usize>, // bytes per second
+
```

Comment:
I'm sure that Alan's inconsistent comment style alarm is going off here 🚨

I'd like to use proper doc comments, because they are highlighted differently in my editor:

<img width="717" alt="Screenshot 2023-09-23 at 11 20 40 AM" src="https://github.com/oxidecomputer/crucible/assets/745333/0f007bd3-d6e2-4bc8-8319-2e5d288bf8e3">

Should I go through and edit the rest of this struct, so it's locally consistent?  Alternatively, I could use `/**` comments here, which are also doc comments (but not recommended by the Rust style guide).

Reply:
😱

Reply:
Okay..
Here are my thoughts.

Change is coming.  The tide is rising.
I think updating just the comment for the field you change in a struct with other comment styles is
the thing to do here.

If/when we make a mass conversion, I think doing it all at once, with one single change to all files with
only changes to comments could be the way to do it.  The idea being that we don't mask comment 
changes with other changes when looking back through git history.  This is just my idea and we should
discuss it further.

`The measure of intelligence is the ability to change` -Albert Einstein
`The secret of change is to focus all of your energy not on fighting the old, but on building the new` -Socrates
`Our dilemma is that we hate change and love it at the same time; what we really want is for things to remain the same but get better.` -Sydney J. Harris
`Change is the law of life, and those who look only to the past and present are certain to miss the future` -John F. Kennedy